common/src/main/java/tachyon/conf/Utils.java

```diff
@@ -97,4 +97,8 @@ public static String getProperty(String property, String defaultValue) {
     LOG.debug("{} {} : {}", property, msg, ret);
     return ret;
   }
+
+  private Utils() {
```

Comment:
``private Utils() {} // prevent instantiation`` is fewer lines of code and more informative